.github/workflows/go-check.yaml

```diff
@@ -28,9 +28,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v1.2.1
+        uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.27
+          version: v1.31
```

Comment:
Just a note on this, golangci-lint-action didn't support patch version (i.e. v1.31 means the latest in v1.31 branch), I think I discussed with the team to support patch version here as well, hope it will get moving soon.

Reply:
The example in https://github.com/golangci/golangci-lint-action/blob/master/README.md says `# Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.` so I assumed this will pick the latest `v1.31` patch version.

Reply:
Yes, few golangci-lint users are asking for patch version in golangci-lint-action to have the same behavior, I am kind of supporting it, the discussion sometimes was indecisive, so I think I just proceed to send PR later :smile: